conf/application.conf

```diff
@@ -164,35 +164,66 @@ versioning {
 
   scopes {
-    "dummy-scope" {
-      fields = [ "A","B","C","D","E","F","G","H","I","J","K","L" ]
+    "read:individuals-benefits-and-credits-laa-c1" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
-    "read:hello-scopes-1" {
-      fields = [ "A","B","C"]
+    "read:individuals-benefits-and-credits-laa-c2" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
-    "read:hello-scopes-2" {
-      fields = [ "A","B"]
+    "read:individuals-benefits-and-credits-laa-c3" {
+        fields = ["B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-hmcts-c2" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-hmcts-c3" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-lsani-c1" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
+    }
+    "read:individuals-benefits-and-credits-lsani-c3" {
+        fields = ["A", "B", "M", "D", "E", "F", "G", "N", "S", "H", "C", "O", "U", "I", "T", "P", "J", "K", "Q", "L", "R"]
     }
   }
 
   endpoints {
-    paye {
-      endpoint = "/individuals/employments/paye?matchId=$randomMatchId{&fromDate,toDate}"
-      fields {
-        A = "payments"
-        B = "employer/employerName"
-        C = "employer/employerAddress/line1"
-        D = "employer/employerAddress/line2"
-        E = "employer/employerAddress/line3"
-        F = "employer/employerDistrictNumber"
-        G = "employer/employerSchemeReference"
-        H = "employmentStartDate"
-        I = "employmentLeavingDate"
-        J = "employmentPayFrequency"
-        K = "employeeAddress"
-        L = "payrollId"
+    benefitsAndCredits {
+        endpoint = "individuals/benefits-and-credits"
+        fields = {
+            A = "applications/id"
+            B = "applications/awards/totalEntitlement"
+            C = "applications/awards/payProfCalcDate"
+          }
+      }
+      childTaxCredits {
+        endpoint = "/individuals/benefits-and-credits/child-tax-credit"
+        fields {
+            D = "applications/awards/childTaxCredit/childCareAmount"
+            E = "applications/awards/childTaxCredit/ctcChildAmount"
+            F = "applications/awards/childTaxCredit/familyAmount"
+            G = "applications/awards/childTaxCredit/babyAmount"
+            H = "applications/awards/childTaxCredit/paidYTD"
+            I = "applications/awards/payments/amount"
+            J = "applications/awards/payments/frequency"
+            K = "applications/awards/payments/startDate"
```

Comment:
You might be able to explain this better as you have looked into this however; why do we have duplicate paths with different keys in each case? 

I.e. 

K = "applications/awards/payments/startDate"

and

Q = "applications/awards/payments/startDate"